java/code/src/com/redhat/rhn/domain/formula/FormulaFactory.java

```diff
@@ -362,6 +362,16 @@ else if (!SystemManager.hasEntitlement(s.getId(), EntitlementManager.MONITORING)
         return orderFormulas(formulas);
     }
 
+    /**
+     * Returns the formulas applied to a given server
```

Comment:
I think this should actually say "assigned" to a given server (or even better "system"), but I know that it is the case also with the above `getFormulasByMinionId()` so no urgent need to change it.

Reply:
Right, I'll change it. Have to do rebase anyway due to changes conflict.

Reply:
Resolved, the respective code was obsolete and meanwhile removed from the patch.